tests/cmdline/commands/test_profile.py

```diff
@@ -162,6 +162,35 @@ def test_delete_force(run_cli_command, mock_profiles, pg_test_cluster):
     assert 'When the `-f/--force` flag is used either `--delete-data` or `--keep-data`' in result.output
 
 
+@pytest.mark.parametrize('entry_point', ('core.sqlite_dos', 'core.sqlite_zip'))
+def test_setup_with_validating_sqlite_version(
+    config_psql_dos, run_cli_command, isolated_config, tmp_path, entry_point, monkeypatch
+):
+    """Test the ``verdi profile setup`` command.
+    Same as `test_setup`, here we check the functionality to check sqlite versions, before setting up profiles.
+    Note that this test should be run before the `test_delete_storage` test.
+    """
+
+    if entry_point == 'core.sqlite_zip':
+        tmp_path = tmp_path / 'archive.aiida'
+        create_archive([], filename=tmp_path)
+
+    profile_name = 'temp-profile'
+    options = [entry_point, '-n', '--profile-name', profile_name, '--email', 'email@host', '--filepath', str(tmp_path)]
+
+    # Should raise if installed version is lower than the supported one.
+    monkeypatch.setattr('aiida.storage.sqlite_zip.backend.SUPPORTED_VERSION', '100.0.0')
+    result = run_cli_command(cmd_profile.profile_setup, options, use_subprocess=False, raises=True)
+    # assert f'Storage backend requires sqlite 100.0.0 or higher. But you have' in result.stderr
```

Comment:
Please add the check that actually checks the message

Reply:
Done, sorry it was commented out by mistake. 
Thanks for checking!